tinygrad/runtime/ops_gpu.py

```diff
@@ -92,7 +91,5 @@ class CLCodegen(CStyleCodegen):
     half_prekernel = "#pragma OPENCL EXTENSION cl_khr_fp16 : enable",
     barrier = "barrier(CLK_LOCAL_MEM_FENCE);", float4 = "(float4)",
     gid = [f'get_global_id({i})' for i in range(3)], lid = [f'get_local_id({i})' for i in range(3)], uses_vload=True)
-  supports_float4_alu = True
```

Comment:
Why was this removed?

Reply:
The parent class defaults these to `True` currently, so behavior is unchanged: https://github.com/geohot/tinygrad/blob/master/tinygrad/codegen/cstyle.py#L182-L183

I figured it was worth the tradeoff to leave these off to save a couple LOC, but can def. add them back in if you'd prefer the explicitness.

Reply:
Nah, didn't realize they were true. Removing is fine

Reply:
Btw, this project doesn't care about LOC anymore, it values readability.

Reply:
I looked into adding a test.  Found one test that seemed like it was applicable at `test/external/external_test_image.py` but it's not currently working due to depending on removed `CLImage` class.

I'll look fixing this a bit more, but would probably be suited for a separate PR at this point.